esmvaltool/namelists/namelist_miles.yml

```diff
@@ -0,0 +1,52 @@
+###############################################################################
```

Comment:
This file should be renamed to recipe_miles.yml

Reply:
Also, please add a documentation section instead of doing documentation in a comment at the top.

Reply:
Or is this file outdated? It does not look like the diagnostic script it calls exists. In that case, it should be removed.

Reply:
Yes, it was an old remnant, I have actually removed it yesterday in https://github.com/ESMValGroup/ESMValTool/pull/735/commits/ddaa5e468b3ee245be67d0bddcda6a4010d24ba3
The recipes are `recipes/recipe_miles_block.yml `and `recipes/recipe_miles_eof.yml`